### ISSUE TITLE
Lack of API 

### ISSUE THREAD

**Issue by Bibhukiju** [Git-Harshit/Random_Text_Generator-Brython#4]
### Lack of api
If there was api 
mobile app could be possible

**Comment by Git-Harshit** [comment]
Yes @Bibhukiju, an API could be included in the tool. It is indeed a good suggestion, thank you for raising this issue!
